modelplugin/plproxy-1.0.0/Makefile

```diff
@@ -0,0 +1,22 @@
+redo: clean copyin models openapi-gen plproxy-3.0.0-openapi3.yaml copyout
+
+copyin:
+#	rm -rf yang
+#	cp -a ../../../sdran-helm-charts/config-models/plproxy-3.x/files/yang .
+
+models:
+	go generate
+	pyang -f tree ../../../roc-helm-charts/config-models/plproxy-1.x/files/yang/*.yang > plproxy-1.0.0.tree
```

Comment:
i think you should the `OPENCORD` variable like we did for the other ones 

Different people keep their `opencord/roc-helm-charts` repo in different places not just `../../..` from here

Reply:
I see OPENCORD was removed in this commit 246ac33bf5ce1c83989d9961eaf7c6b677df453c